dracut/30ignition/ignition-generator

```diff
@@ -32,6 +32,11 @@ add_requires() {
     ln -sf "../${name}" "${requires_dir}/${name}"
 }
 
+# exit early if the installer is active
+if [[ $(cmdline_arg coreos.inst) == "yes" ]]; then
+    exit 0
```

Comment:
Early exit is probably okay on the `spec2x` branch.  On master I'd rather see the test added to the conditional that guards enabling the Ignition target.  Otherwise we could get subtle bugs if other unconditional logic is added further down the script.

Reply:
Yeah, this will look much nicer on master which has a cleaner generator. I checked already that the installer isn't somehow using `ignition.env` (the only possible thing of value that we're skipping out on right now).